source/usage-examples/findOne.txt

```diff
@@ -5,16 +5,16 @@ Find a Document
 .. default-domain:: mongodb
 
 You can retrieve a single document from your collection with the
-``*Collection.FindOne()`` method. 
+``Collection.FindOne()`` method. 
 
 ..
    method
 
-The ``FindOne()`` method takes a query document that specifies the
-subset of your collection that you would like to retrieve a document from. If you
-pass the ``FindOne()`` method an empty document, MongoDB will consider your
-entire collection when retrieving your document. For more information on documents
-in the MongoDB Go driver, see our
+The ``FindOne()`` method takes a filter document. Calling the ``FindOne()``
+method retrieves a document from your MongoDB instance that matches your
```

Comment:
Curious, is "MongoDB instance" docs the standard way to describe a cluster / deployment of MongoDB servers?

Reply:
Interestingly, there is some variance in the terminology used for a MongoDB cluster/instance/deployment across all of our documentation. You will see all of the following throughout Drivers/Server documentation:

- MongoDB deployment ([link](https://docs.mongodb.com/manual/))
- MongoDB instance ([link](https://docs.mongodb.com/manual/mongo/))
- MongoDB ([staging link that may not work in a few weeks](https://docs-mongodbcom-staging.corp.mongodb.com/d7956eb/java/docsworker-xlarge/DOCSP-9572-Logging/fundamentals/enterprise-auth/))
- MongoDB Cluster ([staging link](https://docs-mongodbcom-staging.corp.mongodb.com/d7956eb/java/docsworker-xlarge/DOCSP-9572-Logging/quick-start/))